OutaTime/OutaTime/ViewControllers/DatePickerViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  DatePickerViewController.swift
+//  OutaTime
+//
+//  Created by Cora Jacobson on 6/8/20.
+//  Copyright © 2020 Cora Jacobson. All rights reserved.
+//
+
+import UIKit
+
+protocol DatePickerDelegate {
+    func destinationDateWasChosen(_ date: Date)
+}
+
+class DatePickerViewController: UIViewController {
+
+    @IBOutlet weak var destinationDatePicker: UIDatePicker!
+    
+    var delegate: DatePickerDelegate?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
```

Comment:
I like to delete unused boilerplate like this. You can always add these methods back in, but if you aren't using them, it's best not to have them clog things up visually.